zkevm-circuits/src/evm_circuit/util/math_gadget.rs

```diff
@@ -896,3 +901,255 @@ impl<F: Field> MulAddWordsGadget<F> {
         self.overflow.clone()
     }
 }
+
+#[derive(Clone, Debug)]
+pub(crate) struct ShrWordsGadget<F> {
```

Comment:
Perhaps add a short description what this gadget does?

Reply:
Added a comment `Construction of word shift right for a >> shift == b.`.